docs/topics/eclipse-accessing-tools.adoc

```diff
@@ -6,7 +6,7 @@
 [id="eclipse-accessing-tools_{context}"]
 = Accessing the {ProductShortName} tools
 
-You can access the {PluginName} tools in the *MTA* perspective.
+You can access the {PluginName} tools in the *{ProductShortName}* perspective.
 
```

Comment:
Verify with QE that the name has been chnaged in the IDEs.

Reply:
@RichardHoch  Which name?